Gloss of top docs pages (#28891)
hashicorp/terraform: website/docs/language/data-sources/index.html.md

# Data Sources

> _Data sources_ allow Terraform to fetch or compute data to use elsewhere
> in Terraform configuration. Data sources allows a Terraform
> configuration to use information defined outside of Terraform,
> or defined by another separate Terraform configuration.

Reviewer: The second sentence seems redundant, so combined the two 

```suggestion
_Data sources_ allow Terraform use information defined outside of Terraform,
defined by another separate Terraform configuration, or modified by functions. 
```
